<launch>
     <!-- 乌龟节点 -->
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
    
    <!-- launch-prefix=xterm -e gdb - - args -->
    <node pkg="learning_debug" type="debug_test" name="debug_test" output="screen" >
        <param name = "test_int" type = "int" value = "20" />
        <param name = "test_string" type = "str" value = "dlonng" />
    </node>
</launch>